modules/ROOT/pages/user-management.adoc

```diff
@@ -152,39 +152,304 @@ Each project can have multiple users with individual accounts allowing access to
 The users with access to a project can be viewed and managed from the **Users** page.
 You can access the **Users** page by selecting **Users** from the sidebar menu of the console.
 
+[[roles]]
 === Roles
 
 Users within a project can be assigned one of the following roles:
 
-* _Project Admin_
-* _Project Member_
+* _Data Viewer_
 * _Project Viewer_
+* _Project Member_
+* _Project Admin_
 * _Metrics Reader_
```

Comment:
are these meant to correspond to the roles in table 2? I'm thinking not, but I'm not sure what metrics reader is.